.appveyor.yml

```diff
@@ -2,9 +2,8 @@ init:
   - git config --global core.autocrlf true
 branches:
   only:
-    - master
-    - release
     - dev
+    - /^release\/.*$/
```

Comment:
This need to be on the release/2.1 branch I believe, before appveyor/travis will discover it.